### ISSUE TITLE
User cannot see history in "Process Log"

### ISSUE THREAD

**Issue by ziggurad** [elements-at/ProcessManager#117]
Hello, 

I am using version v3.1.6.

Users in my project who do not have administrator rights cannot see the logs of tasks they have started themselves. Another user (with administrator rights) can see the history.

Can anyone tell what to do users have seen the history of their tasks?

Regards

**Comment by ckogler-elements** [comment]
Hi,
there might be an issue when you  use none numeric Ids for the configruations. Then an error occured (for non admin users only) and the user didn't see the logs. This is fixed in the latest v3.x and v4.x versions.

Regards 
Christian

**Comment by KatrinAckermannEk** [comment]
Are you sure this has been fixed? I still have the issue in version 3.1.13 despite using numeric IDs.

When requesting /admin/elementsprocessmanager/monitoring-item/list:

`Doctrine\DBAL\Exception\InvalidFieldNameException
An exception occurred while executing 'SELECT COUNT(*) as amount FROM plugin_process_manager_monitoring_item  WHERE  published=1  AND  configurationId IN(Array)':

SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Array' in 'where clause'`


Whenever a user is not admin but the role has permission for the process manager and the commands are restricted to all roles, none of the roles or the role of the user (I tested every combination^^).
